editor/src/components/documentation/getting-started.tsx

```diff
@@ -270,15 +270,15 @@ export const GettingStarted = betterReactMemo('Getting Started', () => {
 
       <H1>Change Log - June 2020</H1>
       <div>
-        <p style={{ marginTop: 36 }}>
-          <EM>Highlights of June</EM>
-          <ul style={{ paddingLeft: 30 }}></ul>
-        </p>
+        <P>
+            <EM>Highlights of June</EM>
```

Comment:
looks like prettier isn't set up properly 🤔. it should have automatically removed these extra indents, perhaps someone on @concrete-utopia/maintaineers can help set up tomorrow.

Reply:
Ah I know what the issue here was - when Omar committed this change he was doing it from a different shell that wasn't correctly set up, so the commit hooks didn't run.